### ISSUE TITLE
[a11y][MultiComboBox] - Multi Combo box focus is not visible with keyboard navigation in mobile mode

### ISSUE THREAD

**Issue by yoganjan** [SAP/ui5-webcomponents#8198]
### Bug Description

Multi Combo box focus is not visible in mobile devices when we navigate with the bluetooth enabled keyboard using tabbing. 
This is can be reproducible even in desktop chrome browser with mobile view in Chrom Dev tools

### Affected Component

MultiComboBox

### Steps to Reproduce

1. Open https://sap.github.io/ui5-webcomponents-react/?path=/docs/inputs-multicombobox--docs
2. Change to mobile mode from chorme dev tools using Toogle Device Tool bar and select Iphone 14 Max
3. Try to navigate to MultiComobox using key board tabbing
4. Observe that the focus is not visible

### Priority

None

### UI5 Web Components Version

1.24.0

### Browser

Chrome

### Organization

SAP

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by petyabegovska** [comment]
Note: This is part of bigger topic for visual focus on mobile: FIORITECHP1-17583.

**Comment by Todor-ads** [comment]
Hello @SAP/ui5-webcomponents-topic-rl ,
can you look into this problem?
Тhe issue reproduces after following the steps to reproduce.

Best regards,
Todor

**Comment by petyabegovska** [comment]
Related to: #8250

**Comment by elenastoyanovaa** [comment]
Hello team,

As discussed this will be processed as a feature request.

Kind Regards,
Elena

**Comment by hristop** [comment]
Internal BLI created: BGSOFUIRILA-3812
